### DISCUSSION TITLE
table_name has some error.

### DISCUSSION THREAD

**Discussion by deatil** [SeaQL/sea-orm#1795] [Ideas]
code is:
https://github.com/SeaQL/sea-orm/blob/178f5907f1696f90dce9279a24b67cfb9291eda8/src/entity/active_model.rs#L574

it can change 'fn table_name(&self) -> String'?

i want to use dyn change table_name but now fn is cannot dyn change to return &str. rust will panic cannot return owned data

**Comment by tyt2y3** [comment]
May be a duplicate to https://github.com/SeaQL/sea-orm/discussions/1202

**Reply by deatil** [reply]
it change 'to String' and i think it can work when use dyn name. The give issue cannot use dyn name
